nextcloud_integration/migrations/0001_initial.py

```diff
@@ -1,34 +1,46 @@
-# Generated by Django 2.2.25 on 2022-09-22 21:08
+# Generated by Django 3.2.16 on 2023-01-08 16:00
 
 from django.db import migrations, models
 import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
+    replaces = [('nextcloud_integration', '0001_initial'), ('nextcloud_integration', '0002_auto_20220925_1758'), ('nextcloud_integration', '0003_ncfolder_slug'), ('nextcloud_integration', '0004_auto_20221001_1612'), ('nextcloud_integration', '0005_auto_20221012_1618'), ('nextcloud_integration', '0006_auto_20221016_1845'), ('nextcloud_integration', '0007_auto_20221107_2228'), ('nextcloud_integration', '0008_auto_20230108_1558'), ('nextcloud_integration', '0009_auto_20230108_1559'), ('nextcloud_integration', '0010_auto_20230108_1642')]
```

Comment:
`replaces` also needs to be removed when the squashed-migration files themselves are also removed. See the last bullet point at the bottom of [this](https://docs.djangoproject.com/en/3.2/topics/migrations/#squashing-migrations) docs section.

>You must then transition the squashed migration to a normal migration by:
> - Deleting all the migration files it replaces.
>- Updating all migrations that depend on the deleted migrations to depend on the squashed migration instead.
>- Removing the **replaces** attribute in the **Migration** class of the squashed migration (this is how Django tells that it is a squashed migration).